### DISCUSSION TITLE
Account hacked/hijacked

### DISCUSSION THREAD

**Discussion by Yaachaka** [community/community#137120] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi team, my account `Yaachaka` is taken by someone else `CollinsRonaldzrm@gmail.com`. That person has removed my email as primary account and made his email primary. Please help me with this. Please check the attached screenshots.

Surprisingly i am still able to login to account through mobile using passkey. But I am not able to update the primary email and the email verification is failing.

**Comment by Yackens** [comment]
It sounds like you're experiencing a serious account security issue where someone has gained unauthorized access to your GitHub account and changed the primary email address. Here’s what I would do if I was you immediately!

1. Contact GitHub Support Directly
Report this issue to GitHub Support as soon as possible. You can reach out to them using the [GitHub Support Contact Form](https://support.github.com/contact). Select the option related to "Account Recovery" or "Security Issue."
Provide them with all relevant details, including:
- Your original email address.
- The email address currently associated with the account (the unauthorized one).
- Any recent activity or changes that you noticed.
- The fact that you can still log in using a mobile passkey.

2. Immediate Actions to Take
Do Not Log Out: Since you still have access through your mobile device, do not log out. Try to use this access to remove the unauthorized email and restore your original email as the primary, if possible.
Change Password: If you can, immediately change your account password. This might prevent further unauthorized access.
Revoke Access Tokens and Sessions: If possible, go to your account settings and revoke any existing access tokens or sessions that might have been created by the unauthorized user.
3. Check for Account Recovery
Look into GitHub's [Account Recovery Process](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-user-account-settings/changing-your-primary-email-address#troubleshooting-primary-email-address-changes) to see if you can trigger a recovery based on your prior email address.

If you found this answer useful, please don't forget to mark it as the accepted answer so more people can see it.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Yaachaka,

Thank you for your patience. The community is unable to assist with nor escalate your issue, but like other users mentioned, [please open a Support ticket for assistance](https://support.github.com/contact).  It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
